helm/gcp-compute-persistent-disk-csi-driver/values.yaml

```diff
@@ -26,4 +26,6 @@ image:
     name: quay.io/giantswarm/csi-snapshotter
     tag: v4.0.1
 
-gcpProject: ""
+workloadIdentityPoolID: ""
+gcpServiceAccount: ""
+gcpIdentityProvider: ""
```

Comment:
I have this on the external-dns app. What do you think?

```
# gcpServiceAccount
# The GCP Service Account ID to use when running on gcp, using workload identity for authentication.
gcpServiceAccount: ""

# gcpWorkloadIdentityPoolID
# The workload identity pool ID for the cluster where the app will run. It's of the form '$gcpProject.svc.id.goog'.
gcpWorkloadIdentityPoolID: ""

# gcpIdentityProvider
# The workload identity identity provider for the cluster where the app will run.
gcpIdentityProvider: ""
```

Reply:
Yeah I think that's good. If you convert it to a suggestion I can commit it from here